app/uk/gov/hmrc/pensionschemereturn/transformations/nonsipp/MemberPaymentsTransformer.scala

```diff
@@ -104,6 +107,11 @@ class MemberPaymentsTransformer @Inject()(
         transfersOut <- member.memberTransfersOut.toList.flatten.traverse(transferOutTransformer.fromEtmp)
         pensionSurrenders <- member.memberPensionSurrender.toList.flatten.traverse(pensionSurrenderTransformer.fromEtmp)
       } yield MemberDetails(
+        state = member.memberStatus match {
+          case SectionStatus.New => MemberState.Active
+          case SectionStatus.Changed => MemberState.Active
```

Comment:
I'm not sure how we're handling SectionStatus.Changed in transformToEtmp ? Not sure this is gonna be required for PSR login on our side